src/bp/ui-studio/src/web/components/Layout/Sidebar.jsx

```diff
@@ -81,14 +89,49 @@ class Sidebar extends React.Component {
         <i className="icon material-icons">{module.menuIcon}</i>
       )
 
-    return (
-      <li key={`menu_module_${module.name}`}>
-        <NavLink to={path} title={module.menuText} activeClassName={style.active}>
-          {moduleIcon}
-          <span>{module.menuText}</span>
-        </NavLink>
-      </li>
-    )
+    if (module.name === 'nlu') {
```

Comment:
What do you think of keeping the logic global and allow any module to have submenus ? I'm sure we'll have other modules that will require it, and doing this will allow us to add menu items directly in the module instead of having to release the studio

Reply:
We'll indeed have more modules/menus with submenus. Let's make it a general logic

Reply:
I've added a todo. I don't think we should spend time on this until we actually add another submenu. When that happens, I'll handle it.